spec/requests/forms/delete_confirmation_controller_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 RSpec.describe Forms::DeleteConfirmationController, type: :request do
   let(:form) { build(:form, :with_active_resource, id: 2) }
+  let(:page) { build(:page, form_id: form.id) }
```

Comment:
Thanks for adding tests to the delete confirmation controller! Would you mind putting them in a separate commit?

Reply:
Sorted!